app/models/solidus/gateway/braintree_gateway.rb

```diff
@@ -7,6 +7,7 @@ class Gateway::BraintreeGateway < ::Spree::Gateway
     preference :public_key, :string
     preference :private_key, :string
     preference :always_send_bill_address, :boolean, default: false
+    preference :transmit_shipping_address, :boolean, default: true
```

Comment:
These two bits of configuration having very different names is confusing - is it possible to update the old preference name, possibly with a deprecation warning?